pkg/build/controller/buildconfig/buildconfig_controller.go

```diff
@@ -135,18 +135,18 @@ func (c *BuildConfigController) handleBuildConfig(bc *buildapi.BuildConfig) erro
 		if kerrors.IsConflict(err) {
 			instantiateErr = fmt.Errorf("unable to instantiate Build for BuildConfig %s due to a conflicting update: %v", bcDesc(bc), err)
 			utilruntime.HandleError(instantiateErr)
-		} else if buildgenerator.IsFatal(err) || kerrors.IsNotFound(err) || kerrors.IsBadRequest(err) || kerrors.IsForbidden(err) {
```

Comment:
What happened to `IsFatal`?

Reply:
it use generator which is still internal... I think the clean solution would be to extract that helper into a common helper and share it between generator and controller...

Reply:
fixed.